multimodal/src/autogluon/multimodal/learners/base.py

```diff
@@ -161,13 +161,10 @@ def __init__(
         hyperparameters: Optional[dict] = None,
```

Comment:
Unrelated to this line, but do we want to add a default `eval_metric` for this problem type?

Reply:
Good call! We can add the accuracy as the default.

Reply:
Added accuracy as the default and tested in unit tests.